src/components/configure/keycodes/Keycodes.tsx

```diff
@@ -72,12 +72,12 @@ export default class Keycodes extends React.Component<KeycodesProps, OwnState> {
 
     const keywordSortKeys = keywordKeys.sort((k0, k1) => {
       const index0 = k0.keymap.keycodeInfo.keywords.reduce(
-        (i, kwd) => Math.max(i, kwd.indexOf(search)),
-        -1
+        (i, kwd) => Math.min(i, kwd.indexOf(search)),
+        0
       );
       const index1 = k1.keymap.keycodeInfo.keywords.reduce(
-        (i, kwd) => Math.max(i, kwd.indexOf(search)),
-        -1
+        (i, kwd) => Math.min(i, kwd.indexOf(search)),
+        0
       );
```

Comment:
:+1: The `keywordKeys` absolutely contains keywords which were matched with the search keyword a user fills in. Therefore, the result value of each `reduce` never be -1.